plenum/common/ledger_manager.py

```diff
@@ -661,6 +687,11 @@ def canStartCatchUpProcess(self, ledgerId: int):
                 adjustedQuorum.f + 1:
             # At least once correct node believes that this node is behind.
 
+            # Stop last consistensy proof requesting.
+            if ledgerId in self.consistency_proof_action_ids:
```

Comment:
We should also cancel re-asking for LedgerStatuses at this moment.

Reply:
Done.